docs/spec_draft.md

```diff
@@ -1280,6 +1281,57 @@ For boolean element type, the behavior is same as [stablehlo.or](#stablehloor).
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.map
+
+### Semantics
+
+Performs a map function `computation` over the `inputs` along the `dimensions`
+and produces a `result` tensor where each element is the result of `computation`
+applied to the corresponding elements in `inputs`.
+
+More formally, `result[i0, ..., iR-1] = computation(inputs0[i0, ..., iR-1], `
+`..., inputsN-1[i0, ..., iR-1])`.
+
+### Inputs
+
+| Name          | Type                                             |
+|---------------|--------------------------------------------------|
+| `inputs`      | variadic number of tensors of any supported type |
+| `dimensions`  | 1-dimensional tensor constant of type `si64`     |
```

Comment:
Can we drop  `dimensions` if not relevant?

Reply:
This would require a change in ODS. I created #487 to track it.